.gitignore

```
@@ -48,3 +55,23 @@ build-iPhoneSimulator/
 
 # unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
 .rvmrc
+
+# Ignore master key for decrypting credentials and more.
+/config/master.key
+
+config/settings.local.yml
+config/settings/*.local.yml
+config/environments/*.local.yml
+config/credentials.yml.enc.example
```

```suggestion
config/credentials.yml.enc
```
ignore file gốc để example lại cơ mà em